src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -1,44 +1,67 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
+import Tooltip from 'app/components/tooltip';
 import {IconCheckmark, IconFire, IconWarning} from 'app/icons';
-import {t} from 'app/locale';
+import {t, tct} from 'app/locale';
 import space from 'app/styles/space';
+import {WebVital} from 'app/utils/discover/fields';
 import {formatPercentage} from 'app/utils/formatters';
 import theme, {Color} from 'app/utils/theme';
 
-import {VitalState, vitalStateColors} from './utils';
+import {VitalState, vitalStateColors, webVitalMeh, webVitalPoor} from './utils';
 
 type Percent = {
   vitalState: VitalState;
   percent: number;
 };
 
 type Props = {
+  vital: WebVital | WebVital[];
   percents: Percent[];
   showVitalPercentNames?: boolean;
 };
 
+const vitalStateIcons = {
+  [VitalState.POOR]: IconFire,
+  [VitalState.MEH]: IconWarning,
+  [VitalState.GOOD]: IconCheckmark,
+};
+
+function vitalStateText(vital, vitalState) {
```

Comment:
Personal pref here but I thinking adding a verb here would give a bit more context of what it's doing, e.g. `getVitalStateText`